sumpy/e2p.py

```diff
@@ -322,15 +315,15 @@ def get_optimized_kernel(self):
                 enforce_variable_access_ordered="no_check")
         return knl
 
-    def __call__(self, queue, **kwargs):
-        knl = self.get_cached_optimized_kernel()
```

Comment:
Can you keep this line? i.e.
```
knl = self.get_cached_optimized_kernel()
return actx.call_loopy(knl, centers=centers, rscale=rscale, **kwargs)
```
This might be a nitpick, but it saves a few keystrokes everytime I debug these kernels.

Reply:
The call is still there in the lines below, just inlined in the `call_loopy`. Did you mean you'd prefer to have the kernel construction on a separate line to be able to break on it?

Reply:
Yes, that's what I mean. Thanks